card/db/migrate_core_cards/data/cards/shark_menu

```diff
@@ -0,0 +1,2 @@
+<p><strong>[[http://decko.org/Features| features]]</strong>: [[http://decko.org/cards|cards]], [[http://decko.org/types|types]], [[http://decko.org/inclusions|inclusions]], [[http://decko.org/views|views]], [[http://decko.org/rules|rules]], [[http://decko.org/layouts|layouts]], [[http://decko.org/formatting|formatting]], [[http://decko.org/accounts|accounts]], [[http://decko.org/permissions|permissions]]</p>
```

Comment:
Not worth much energy, but there may be benefits to using https instead of http. (Performance cost of redirect is miniscule, but maybe SEO?)